### PR TITLE
rename transports

### PR DESCRIPTION
**Pull request by BertKleewein**
# Description of the problem
provisioning transports were named incorrectly according to the visio file.  Easy fix now, much harder fix later.

### CODE REVIEW

File: provisioning/transport/amqp/package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "azure-iot-provisioning-amqp",
+  "name": "azure-device-provisioning-amqp",
```
Comment (anthonyvercolano) [inline provisioning/transport/amqp/package.json]: >azure-device-provisioning-amqp [](start = 11, length = 30)

I think it should be azure-provisioning-device-client

File: provisioning/transport/amqp/package.json [hunk 2]
```diff
@@ -9,6 +9,7 @@
   "dependencies": {
     "azure-iot-amqp-base": "1.2.1",
     "azure-iot-common": "1.2.1",
+    "azure-device-provisioning-client": "0.0.1",
```
Comment (anthonyvercolano) [inline provisioning/transport/amqp/package.json]: >azure-device-provisioning-client [](start = 5, length = 32)

azure-provisioning-device-client

File: provisioning/transport/http/package.json [hunk 3]
```diff
@@ -9,6 +9,7 @@
   "dependencies": {
     "azure-iot-http-base": "1.2.0",
     "azure-iot-common": "1.2.1",
+    "azure-device-provisioning-client": "0.0.1",
```
Comment (anthonyvercolano) [inline provisioning/transport/http/package.json]: See comment for amqp.

File: provisioning/transport/mqtt/package.json [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "azure-iot-provisioning-mqtt",
+  "name": "azure-device-provisioning-mqtt",
```
Comment (anthonyvercolano) [inline provisioning/transport/mqtt/package.json]: >ice-provisioning-mqtt", [](start = 20, length = 23)

see comment for http.

File: provisioning/transport/amqp/package.json (hunk 1 above)
Comment (pierreca) [reply provisioning/transport/amqp/package.json]: really? "Azure provisioning" carries a very different meaning than "azure device provisioning". I'm really not sure about that one...

---
In reply to: [147511225](https://github.com/Azure/azure-iot-sdk-node/pull/126#discussion_r147511225) [](ancestors = 147511225)

File: provisioning/transport/amqp/package.json [hunk 5]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "azure-iot-provisioning-amqp",
+  "name": "azure-device-provisioning-amqp",
   "version": "0.0.1",
```
Comment (pierreca) [inline provisioning/transport/amqp/package.json]: Please do not check in without reviewing all dev-setup/teardown scripts

**Pull request closed without merging**